spec/request_helper.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module Linzer
+  module Test
+    module Request
+      extend self
+
+      def new_request(verb, uri = "/", params = {}, headers = {})
```

Comment:
`uri` must be a path though, so maybe take this opportunity to rename the parameter?

Reply:
Fixed, thanks for the reminder.